src/Compilers/CSharp/Portable/Binder/SwitchBinder_BindPatternSwitch.cs

```diff
@@ -166,21 +166,25 @@ private SourceLabelSymbol FindMatchingSwitchLabel(object key)
             return map;
         }
 
-        private ImmutableArray<BoundPatternSwitchSection> BindPatternSwitchSections(BoundExpression boundSwitchExpression, SyntaxList<SwitchSectionSyntax> sections, Binder originalBinder, ref DefaultSwitchLabelSyntax defaultLabel, DiagnosticBag diagnostics)
+        private ImmutableArray<BoundPatternSwitchSection> BindPatternSwitchSections(BoundExpression boundSwitchExpression, SyntaxList<SwitchSectionSyntax> sections, Binder originalBinder, ref BoundPatternSwitchLabel defaultLabel, DiagnosticBag diagnostics)
```

Comment:
I did this to be consistent with the way it is handled in `BindPatternSwitchSection`, but an `out` parameter (initialized to null here rather than in the caller) would work too. I'll change it.

---
In reply to: [69157587](https://github.com/dotnet/roslyn/pull/12250#discussion_r69157587) [](ancestors = 69157587)